### DISCUSSION TITLE
deck.gl-community/react npm package

### DISCUSSION THREAD

**Discussion by rsbh** [visgl/deck.gl-community#105] [General]
issue ref https://github.com/visgl/deck.gl-community/issues/104

Hey, we are using `HtmlOverlay` and `HtmlOverlayItem` from `@nebula.gl/overlays` in our project. As i can see in the [doc](https://visgl.github.io/deck.gl-community/docs/modules/editable-layers#editable-layers-v90) that `@nebula.gl/overlays`  has been migrated to `@deck.gl-community/react` . When is it releasing to npm?

**Comment by ibgreen** [comment]
[Try installing v9.0.1](https://www.npmjs.com/package/@deck.gl-community/react?activeTab=versions)
